Utilities/import-llvm

```diff
@@ -190,6 +235,51 @@ def main():
 
     include_dir = os.path.join(sourcekit_srcroot, 'include')
 
+    # Create Symlinks for SwiftPM include directory resolution
+    os.makedirs(os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm'))
+    os.makedirs(os.path.join(sourcekit_llvm_lib, 'Demangle', 'include'))
+    os.makedirs(os.path.join(sourcekit_llvm_lib, 'include'))
+    os.makedirs(os.path.join(sourcekit_llvm_include, 'Demangle', 'include'))
+    os.symlink(os.path.join('..', '..', '..', '..', '..', 'include', 'llvm', 'ADT'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm', 'ADT'))
+    os.symlink(os.path.join('..', '..', '..', '..', '..', 'include', 'llvm', 'Config'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm', 'Config'))
+    os.symlink(os.path.join('..', '..', '..', '..', '..', 'include', 'llvm', 'Support'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm', 'Support'))
+    os.symlink(os.path.join('..', '..', '..', '..', 'include', 'llvm-c'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm-c'))
+    os.symlink(os.path.join('..', '..', '..', '..', 'include', 'llvm'),
+               os.path.join(sourcekit_llvm_lib, 'Demangle', 'include', 'llvm'))
+    os.symlink(os.path.join('..', '..', '..', 'llvm'),
+               os.path.join(sourcekit_llvm_include, 'Demangle', 'include', 'llvm'))
+    os.symlink(os.path.join('..', '..', '..', 'include', 'llvm'),
+               os.path.join(sourcekit_llvm_lib, 'include', 'llvm'))
+    os.symlink(os.path.join('..', '..', '..', 'include', 'llvm-c'),
+               os.path.join(sourcekit_llvm_lib, 'include', 'llvm-c'))
```

Comment:
Could these symlinks be done by looping over the directories instead of listing them out manually?  It's really hard to tell if these are correct by inspection.

Reply:
Cleaned these up a bit, I didn't need as many as I thought I did.